curriculum/challenges/chinese/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/make-a-person.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: a2f1d72d9b908d0bd72bb9f6
-title: Make a Person
+title: 构建 Person 类
```

Comment:
@miyaliu666 Why is "Person" not translated?

Reply:
Because "person" is an object name, we thought it's ok not translating it. But I've updated the translation to be "创建一个人员对象" in Crowdin, with "person" translated.